sniffles2/2.2/Dockerfile

```diff
@@ -0,0 +1,48 @@
+ARG VERSION=3.0.0
```

Comment:
VERsion should match the main version of the program. In this case 2.2

Reply:
Resolved